### PR TITLE
Update some strings

### PR DESCRIPTION
**Pull request by LWinterberg**
* Resolves: #4972
* Updates an FFmpeg link
* Updates the help box for the project rate
* Updates credits

### CODE REVIEW

File: src/prefs/DevicePrefs.cpp [hunk 1]
```diff
@@ -257,7 +257,7 @@ void DevicePrefs::PopulateOrExchange(ShuttleGui & S)
                auto helpBtn = S.AddBitmapButton(theTheme.Bitmap(bmpHelpIcon));
 
                const auto helpText =
-                  XO("Project Sample Rate used when recording new tracks and for playback, mixdowns and exports in this project")
+                  XO("Sample Rate used when recording new tracks, mixing down tracks and for playback in this project.")
```
Comment (Paul-Licameli) [inline src/prefs/DevicePrefs.cpp:260]: Mix down is understood to include export?

**Review by Paul-Licameli — APPROVED** [review]

File: src/prefs/DevicePrefs.cpp (hunk 1 above)
Comment (LWinterberg) [reply src/prefs/DevicePrefs.cpp:260]: Export now has a sample rate setting on its own and no longer needs to be included here.

File: src/AboutDialog.cpp [hunk 2]
```diff
@@ -128,12 +128,12 @@ void AboutDialog::CreateCreditsList()
       XO("%s, graphics");
 
    // The Audacity Team: developers and support
-   AddCredit(wxT("Gonzalo Guzm\u00E1n"), documentationAndSupportFormat, roleTeamMember);
+   AddCredit(wxT("Matthieu Hodgkinson"), developerFormat, roleTeamMember);
```
Comment (saintmatthieu) [inline src/AboutDialog.cpp:131]: 🤩

**Review by Paul-Licameli — APPROVED** [review]

**Comment by dozzzzer** [comment]
Tested

**Pull request merged**